doc/ledger3.texi

```diff
@@ -6960,9 +6966,13 @@ meaning of the flag (instead of the first five transactions being
 printed, for example, it would print all but the first five).
 
 @item --pager @var{FILE}
-Tell Ledger to pass its output to the given pager program; very useful
-when the output is especially long.  This behavior can be made the
-default by setting the @env{LEDGER_PAGER} environment variable.
+Tell Ledger to pass its output to the given @var{FILE} pager program;
+very useful when the output is especially long.  This behavior can be
+made the default by setting the @env{LEDGER_PAGER} environment variable.
+
+@item --no-pager
+Tell Ledger to @emph{not} pass its output to a pager program; useful
+when a pager is set by default.
```

Comment:
Ou to curiosity I just tested `ledger --no-pager --pager less -f test/input/sample.dat reg` (displays report with `less`) and `ledger --pager less --no-pager -f test/input/sample.dat reg` (displays report directly) it might be worthwhile to document that the "the last given option wins".